### PR TITLE
Use different Docker image for blender verification

### PR DESCRIPTION
**Pull request by Krigpl**
Crops are rendered in the same docker run as the rest of the verification.

All these new scripts are taken from https://gitlab.lan.golem.network/lfoniok/blender-docker/tree/graphene-blender-experiments, in particular, that means I won't be able to answer any questions related to them

### CODE REVIEW

**Comment by Krigpl** [comment]
Also closes #3772

**Comment by Krigpl** [comment]
catping

**Comment by Krigpl** [comment]
ping @magdasta @nieznanysprawiciel

File: apps/blender/resources/images/scripts_verifier/img_metrics_calculator.py [hunk 1]
```diff
@@ -74,114 +72,88 @@ def compare_crop_window(cropped_img_path,
...
-    crops.append(scene_crop_down_left)
 
+def get_crops(rendered_scene, x, y, width, height):
+    crops = []
+    offsets = [0, 1, -1]
+    for x_offset in offsets:
+        for y_offset in offsets:
+            crop = rendered_scene.crop((x + x_offset,
+                                        y + y_offset,
+                                        x + width - x_offset,
```
Comment (Krigpl) [inline apps/blender/resources/images/scripts_verifier/img_metrics_calculator.py]: Isn't this the buggy version? `-` instead of `+` ?

Comment (nieznanysprawiciel) [reply apps/blender/resources/images/scripts_verifier/img_metrics_calculator.py]: Yes it is.
But fix is waiting in different pull request. And previous version wasn't less buggy probably

Comment (nieznanysprawiciel) [reply apps/blender/resources/images/scripts_verifier/img_metrics_calculator.py]: Or I can apply this fix in this pull request.
I don't know

Comment (Krigpl) [reply apps/blender/resources/images/scripts_verifier/img_metrics_calculator.py]: Please apply in this PR, there's no point in submitting known buggy code.

Comment (nieznanysprawiciel) [reply apps/blender/resources/images/scripts_verifier/img_metrics_calculator.py]: ok, done

File: tests/apps/blender/verification/test_verificator_integration.py [hunk 2]
```diff
@@ -1,256 +0,0 @@
-import logging
```
Comment (nieznanysprawiciel) [inline tests/apps/blender/verification/test_verificator_integration.py:1]: Why these tests were removed?

**Review by nieznanysprawiciel — APPROVED** [review]

**Pull request merged**